Tools/Scripts/webkitpy/api_tests/runner.py

```diff
@@ -183,8 +188,9 @@ def __init__(self, port):
     def _filter_noisy_output(cls, output):
         result = ''
         for line in output.splitlines():
-            if line.lstrip().startswith('objc['):
-                continue
+            for expression in cls.NOISY_OUTPUT:
```

Comment:
Why does `cls` refer to `_Worker` here?

Reply:
That's how Python's `classmethod`s work. Worth noting (which is no doubt what prompted this comment) that a previous version of this change had `NOISY_OUTPUT` incorrectly belonging to a different class.